webapp/components/edit_channel_header_modal.jsx

```diff
@@ -179,15 +180,15 @@ class EditChannelHeaderModal extends React.Component {
                         <Textbox
                             value={this.state.header}
                             onChange={this.handleChange}
-                            onKeyPress={this.handleEditKeyPress}
+                            onKeyPress={this.handleKeyPress}
                             onKeyDown={this.handleKeyDown}
                             supportsCommands={false}
                             suggestionListStyle='bottom'
                             createMessage={Utils.localizeMessage('edit_channel_header.editHeader', 'Edit the Channel Header...')}
                             previewMessageLink={Utils.localizeMessage('edit_channel_header.previewHeader', 'Edit Header')}
                             handlePostError={this.handlePostError}
                             id='edit_textbox'
-                            ref='textbox'
+                            ref='editChannelHeaderTextbox'
```

Comment:
FYI, if you changed the ref so that it was different from other components with a ref named `textbox`, that's not necessary. Refs only need to be unique within the component